library/registry

```diff
@@ -1,8 +1,11 @@
 Maintainers: Milos Gajdos (@milosgajdos),
              Sebastiaan van Stijn (@thajeztah)
 GitRepo: https://github.com/docker/distribution-library-image.git
+
+Tags: 3.0.0-alpha.1
+Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
 GitFetch: refs/heads/master
```

Comment:
This should leave `GitFetch:` up in the global block (so it applies to all entries in the file):

```suggestion
GitFetch: refs/heads/master

Tags: 3.0.0-alpha.1
Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
```